vFile: Initialization failed for docker version 17.06.0-ce

It is mentioned in the documentation here http://vmware.github.io/docker-volume-vsphere/documentation/vfile-plugin.html that docker version 17.06.0 or newer is supported by vFile. 
Yet, the initialization failed for docker version ```17.06.0-ce```.

Logs:
```
2017-12-05 16:36:54.718739375 -0800 PST [ERROR] Current docker version 17.06.0-ce doesn't satisfy required version 17.06
```

--- a/client_plugin/drivers/vfile/dockerops/dockerops.go
+++ b/client_plugin/drivers/vfile/dockerops/dockerops.go
@@ -577,7 +577,14 @@ func (d *DockerOps) CheckDockerVersion(requiredVersion string) (bool, error) {
 		log.Errorf("Failed to get docker server version. Error: %v", err)
 		return false, err
 	}
-	v1, err := version.NewVersion(serverVersion.Version)
+	// serverVersion.Version read from docker has the format like this
+	// 17.06.0-ce
+	// need to extract the numeric part to compare
+	versionStr := strings.Split(serverVersion.Version, "-")
+	// dockerServersion only has the numeric part "17.06.0"
+	dockerServerVersion := versionStr[0]
+	log.Infof("dockerServerVersion: %s", dockerServerVersion)
+	v1, err := version.NewVersion(dockerServerVersion)
 	if err != nil {
 		log.Errorf("Failed to create version comparison for %s. Error: %v", serverVersion.Version, err)
 		return false, err

--- a/client_plugin/drivers/vfile/dockerops/dockerops.go
+++ b/client_plugin/drivers/vfile/dockerops/dockerops.go
@@ -583,7 +583,7 @@ func (d *DockerOps) CheckDockerVersion(requiredVersion string) (bool, error) {
 	versionStr := strings.Split(serverVersion.Version, "-")
 	// dockerServersion only has the numeric part "17.06.0"
 	dockerServerVersion := versionStr[0]
-	log.Infof("dockerServerVersion: %s", dockerServerVersion)
+	log.Debugf("dockerServerVersion: %s", dockerServerVersion)
 	v1, err := version.NewVersion(dockerServerVersion)
 	if err != nil {
 		log.Errorf("Failed to create version comparison for %s. Error: %v", serverVersion.Version, err)

--- a/client_plugin/drivers/vfile/vfile_driver.go
+++ b/client_plugin/drivers/vfile/vfile_driver.go
@@ -58,7 +58,7 @@ const (
 	initError            = "vFile volume driver is not fully initialized yet."
 	mountError           = "exit status 255"
 	checkTicker          = time.Second
-	requiredVersion      = "17.06"
+	requiredVersion      = "17.06.0"
 )
 
 /* VolumeDriver - vFile plugin volume driver struct

